script/create_dev_conda_env.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+CUDA_VERSIONS="10.2,11.3,11.6,11.7"
+TORCH_VERSION="1.12.0"
```

Comment:
`1.12.0` or `1.12`? The latest of PyTorch 1.12 is `1.12.1`. If we use `1.12` here, the latest one will be installed?

Reply:
Currently we are using 1.12.0 in our CI, let's update it later.